Dikke Ploaten/Controller/SignupViewController.swift

```diff
@@ -75,7 +75,18 @@ class SignupViewController: UIViewController, UITextFieldDelegate {
 				self.performSegue(withIdentifier: "signupToHome", sender: self)
 			}, failureHandler: { error in
 				// Error creating user
-				let alertController = UIAlertController(title: "Error", message: error.localizedDescription, preferredStyle: .alert)
+				let alertTitle = NSLocalizedString("Whoops", comment: "")
+				var alertMessage = NSLocalizedString("", comment: "")
+				if(error.localizedDescription == "The email address is badly formatted."){
```

Comment:
Niet op `localizedDescription` checken, gebruik de error code of error name (normaal in `userInfo`)